sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/RetryUtil.java

```diff
@@ -22,6 +22,10 @@
  * Helper class to help with retry policies.
  */
 public class RetryUtil {
+    private static final double JITTER_FACTOR = 0.08;
+    // Base sleep wait time.
+    private static final Duration SERVER_BUSY_WAIT_TIME = Duration.ofSeconds(4);
```

Comment:
Does the AMQP response contain any information on backoff when the server is busy and this is a best approximation default?

Reply:
Yes